### PR TITLE
Match bot edit [WAIV-1028]

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/MatchBot/CreateRule/CreateRule.js [hunk 1]
```diff
@@ -1,33 +1,50 @@
 import React, { useEffect, useState } from 'react';
-import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
 import { injectIntl } from 'react-intl';
-import { Button, Form, Input, message, Modal, Slider } from 'antd';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import moment from 'moment';
```
Comment (milklim) [inline src/client/rewards/MatchBot/CreateRule/CreateRule.js]: add package to project

File: src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+const DeleteRuleModal = ({
+  deleteLoading,
+  handleModalVisibility,
+  intl,
+  isDeleteModal,
+  onOk,
+  sponsor,
+}) => {
+  const handleDeleteRule = () => handleModalVisibility();
```
Comment (milklim) [inline src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js]: reason to reassign?

File: src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <div className="DeleteRuleModal__content">
+        {intl.formatMessage({
+          id: 'matchBot_do_you_want_delete_match_bot_rules',
+          defaultMessage: 'Do you want to delete match bot rules for',
+        })}
+        <Link to={`/@${sponsor}`}>{` @${sponsor}`}</Link>?
+      </div>
+      <div className="DeleteRuleModal__footer">
+        <div className="CreateRule__button">
+          <Button onClick={handleModalVisibility} disabled={deleteLoading}>
```
Comment (milklim) [inline src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js]: 'handleModalVisibility' assigned to 'handleDeleteRule' - use one of them

File: src/client/rewards/MatchBot/MatchBotTable/MatchBotTableRow.js [hunk 4]
```diff
@@ -1,41 +1,65 @@
...
 import { setMatchBotRules } from '../../rewardsActions';
+import { formatDate } from '../../rewardsHelper';
+import getMatchBotMessageData from '../matchBotMessageData';
 
-const MatchBotTableRow = ({ intl, rule, handleEditRule }) => {
-  const [modalVisible, setModalVisible] = useState(false);
-  const [isLoading, setLoaded] = useState(false);
+const MatchBotTableRow = ({ handleEditRule, handleSwitcher, isAuthority, intl, rule }) => {
   const [activationStatus, setActivationStatus] = useState('');
+  const [isLoading, setLoaded] = useState(false);
```
Comment (milklim) [inline src/client/rewards/MatchBot/MatchBotTable/MatchBotTableRow.js]: loading of loaded?

File: src/client/rewards/MatchBot/CreateRule/CreateRule.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/MatchBot/CreateRule/CreateRule.js]: ++

File: src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js]: ++

File: src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js (hunk 3 above)
Comment (konstantin1390) [reply src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js]: ++

File: src/client/rewards/MatchBot/MatchBotTable/MatchBotTableRow.js (hunk 4 above)
Comment (konstantin1390) [reply src/client/rewards/MatchBot/MatchBotTable/MatchBotTableRow.js]: ++

**Pull request merged**